### PR TITLE
Adds variety of weapons to wastelanders

### PR DESCRIPTION
**Pull request by Cw3040**
-adds a weapon to the pre-equip for the wastelander role

 :cl:
add: Adds a random melee weapon to the spawning equipment for wastelanders.
   /:cl:

### CODE REVIEW

**Comment by improvedname** [comment]
Changelog required.

File: code/modules/jobs/job_types/wasteland.dm [hunk 1]
```diff
@@ -64,7 +71,7 @@ wasteland
 	backpack_contents = list(
 		/obj/item/reagent_containers/hypospray/medipen/stimpak=2, \
 		/obj/item/reagent_containers/blood/radaway=1, \
-		/obj/item/reagent_containers/food/drinks/flask, \
```
Comment (UndeadNappist) [inline code/modules/jobs/job_types/wasteland.dm]: Leave the flask in. Maybe randomspawn what alcohol or water starts in it.

Comment (Cw3040) [reply code/modules/jobs/job_types/wasteland.dm]: They spawn with two, nappist. The second is universally thrown out.

**Comment by UndeadNappist** [comment]
Closing and then reopening to force travis to rerun.

File: code/modules/jobs/job_types/wasteland.dm [hunk 2]
```diff
@@ -64,7 +71,7 @@ wasteland
...
+		\
```
Comment (UndeadNappist) [inline code/modules/jobs/job_types/wasteland.dm]: This needs to not be here.

**Pull request merged**